test/e2e/page-objects/pages/home/non-evm-homepage.ts

```diff
@@ -47,30 +47,15 @@ class NonEvmHomepage extends HomePage {
     balance: string,
     token: string = 'SOL',
   ): Promise<void> {
-    await this.driver.wait(async () => {
-      try {
-        await this.driver.waitForSelector(
-          {
-            text: balance,
-            tag: 'span',
-          },
-          { timeout: 1000 },
-        );
-        return true;
-      } catch (e) {
-        console.log('Error in check_getBalance', e);
-        await this.driver.refresh();
-        return false;
-      }
-    }, 30000);
-    await this.driver.waitForSelector(
-      {
-        text: token,
-        tag: 'span',
-      },
-      { timeout: 60000 },
-    );
-    await this.driver.refresh();
```

Comment:
This logic should be removed. Instead of refreshing the wallet everytime, we should just wait for the target selectors. This might have been introduced given that responses were not mocked, so it took a significant amount of time for the values to load, but now that we have mocks, we can remove this logic entirely


Before: See a refresh performed

https://github.com/user-attachments/assets/54d78def-0a4d-4f9e-aef2-056eaf57e079

After: no refreshes, just waiting for the target selector

https://github.com/user-attachments/assets/6dd1b459-f915-4522-948e-90eccf8903b2